From a972976b15b089674c669bb1eab4e0ec4d8be0ec Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Tue, 24 Jan 2023 10:01:23 +0100
Subject: [PATCH 03/29] COMP: Adapt to new qRestAPI cmake

This removes the use of the old `qRestAPI_USE_FILE`. As of today, this
is a SystoleOS adaptation that should not reach the official Slicer
repository until commontk/qRestAPI#25 gets merged.
---
 Base/QTCore/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Base/QTCore/CMakeLists.txt b/Base/QTCore/CMakeLists.txt
index 0c9aead4d0..86ee0e25f1 100644
--- a/Base/QTCore/CMakeLists.txt
+++ b/Base/QTCore/CMakeLists.txt
@@ -41,9 +41,8 @@ endif()
 # qRestAPI
 #
 if(Slicer_BUILD_EXTENSIONMANAGER_SUPPORT OR Slicer_BUILD_APPLICATIONUPDATE_SUPPORT)
-  find_package(qRestAPI REQUIRED)
-  include(${qRestAPI_USE_FILE})
-endif()
+ find_package(qRestAPI REQUIRED)
+ endif()
 
 #
 # CTKAppLauncherLib
@@ -60,6 +59,7 @@ set(KIT_export_directive "Q_SLICER_BASE_QTCORE_EXPORT")
 set(KIT_include_directories
   ${MRMLLogic_INCLUDE_DIRS}
   ${MRMLDisplayableManager_INCLUDE_DIRS}
+  ${qRestAPI_INCLUDE_DIRS}
   )
 
 if(Slicer_BUILD_CLI_SUPPORT)
-- 
2.39.2

